webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/operate/template/FlowNodeInstanceTemplate.java

```diff
@@ -25,7 +25,7 @@ public class FlowNodeInstanceTemplate extends OperateTemplateDescriptor
   public static final String PROCESS_INSTANCE_KEY = "processInstanceKey";
   public static final String PROCESS_DEFINITION_KEY = "processDefinitionKey";
   public static final String BPMN_PROCESS_ID = "bpmnProcessId";
-  public static final String INCIDENT_KEY = "incidentKey";
+  @Deprecated public static final String INCIDENT_KEY = "incidentKey";
```

Comment:
❓ Why was this deprecated?

Reply:
Ah 
>deprecated and stopped filling incidentKey field as it's not used for a long time

I think would be good if we have this as a comment and part of a commit message, to make this a bit more clear.

Reply:
We decided to bring it back as it was exposed in REST API v.1